kieserver/image.yaml

```diff
@@ -199,7 +199,31 @@ envs:
     description: "Disable Maven pull dependencies for immutable KIE Server configurations for S2I and pre built kjars. Useful for pre-compiled kjar."
   - name: "KIE_SERVER_DISABLE_KC_VERIFICATION"
     example: "true"
-    description: "Disable Maven KIE Jar verification. It is recommended to test the kjar manually before disabling this verification."
+    description: "Disable Maven KIE Jar verification. It is recommended to test the kjar manually before disabling this verification."  
+  - name: "org.kie.kafka.server.ext.disabled"
```

Comment:
shouldn't the env name be here and the system property on ks image?

Reply:
Updated